### ISSUE TITLE
Error when trying to migrate using git+ssh url

### ISSUE THREAD

**Issue by XANi** [go-gitea/gitea#3008]
![zaznaczenie_2017-11-28_002](https://user-images.githubusercontent.com/147394/33310895-709dafa2-d423-11e7-8f1a-b4740f6a9a8a.png)

- Gitea version (or commit ref): 1.3.0 and whatever is on try.gitea.io
- Can you reproduce the bug at https://try.gitea.io:
  - [X] Yes (screenshot was done at try.gitea.io)
  - [ ] No
  - [ ] Not relevant

changing "Migration type" doesn't matter

using alternative git url like `ssh://git@github.com/zerosvc/go-zerosvc.git` doesn't help. Doing it via HTTPS works but that is not exactly ideal as not every repo migrated from will be accessible via https

**Comment by lunny** [comment]
SSH currently not supported.

**Comment by RickZeeland** [comment]
@XANi see issue #1635 for tips on migrating using Git. 
Also if you want to use SSH you will have to set the SSH/GPG keys in the user menu.
On Windows 10 I also had to copy the SSH* utilities from the Git directory to the Gitea directory, adding to the PATH did not work.
In the app.ini under [server] I had to add:
`START_SSH_SERVER = true
`
More info on SSH here: https://help.github.com/articles/connecting-to-github-with-ssh/
